src/pages/home.js

```diff
@@ -9,27 +9,27 @@ function Home() {
       <h1 className="text-white ml-32 mt-16 text-5xl">
         Welcome to the Space Center
       </h1>
-      <div className="text-gray mt-6 ml-32 w-1/2">
+      <div className="text-gray mt-6 ml-32 w-1/2 font-IBMPlexSansRegular">
         Here you can explore the Subvisual Universe and reach every corner of
         it. Learn about other incredible members of this community by completing
         tasks every week and make new connections. And who knows, maybe it's
         here where you'll find some very good friends.
       </div>
-      <div className="ml-32 mt-32 flex gap-5 h-80 flex-shrink max-sm:flex-col">
-        <Link to="/question">
-          <button className=" max-w-[510px] max-sm:max-w-[255px] rounded-3xl bg-question text-white p-9 h-full bg-cover bg-center">
-            <h1 className="text-base">QUESTION OF THE WEEK</h1>
-            <div className="text-sm">
+      <div className="ml-32 mt-32 flex gap-5 h-80 flex-shrink max-sm:flex-col mr-32">
+        <Link to="/question" className="flex-1">
+          <button className="w-full rounded-3xl bg-question text-white p-9 h-full bg-cover bg-center">
+            <h1 className="text-xl">WEEKLY QUESTION</h1>
```

Comment:
Personally I prefer to write this normally and then use `uppercase`. I believe for screen readers it works best as well, otherwise they interpret it as shouting.

Reply:
Oh ok!